### DISCUSSION TITLE
3/3 Challenge mode tokens - Atiesh cost

### DISCUSSION THREAD

**Discussion by andovcr** [chromiecraft/chromiecraft#5923] [Suggestions]
I propose that Atiesh be reduced from costing 5 tokens to costing 3.  I know this will be met with opposition, but I am hoping to hear the rationale from the devs on why, if they choose to deny this suggestion.

Currently, you can only earn tokens by completing 3/3 Challenge Modes on the same character simultaneously.  This is an incredible effort, as evidenced by the fact that only 5 people have accomplished it so far:
Jesterfail - Hunter
Ponytailhc - Warrior
Hcimbt - Paladin
Confidential - Warlock
Zorgo - Hunter

The tokens can only be traded within the same account, and accounts are limited to 10 characters.  Finally, only Contributors (i.e. people who help with bug triaging, testing, development, GMs, etc…) can move characters to a different account.  If a player has already leveled up multiple toons on a single account, in order to earn Atiesh they could be forced to either delete multiple existing characters, or do the 3/3 challenge once, delete, and then repeat.

Atiesh, as a legendary, was designed to limit the RNG issue facing Sulfuras & Thunderfury by having the player collect splinters and then killing Kel'thuzad, C'thun, & Atiesh.  While still a grind, this created the very real possibility that raids could build an Atiesh staff before ever seeing other highly sought after Naxx weapons (like Corrupted Ashbringer, Might of Menethil, Hungering Cold, etc.).

Currently I believe all of those Naxx weapons cost a single token (regardless of if they are 1H, 2H, or Ranged), with the sole exception of Atiesh costing 5.  I propose that it be reduced to costing 3 tokens, which still elevates it from the other weapons by costing more, but is more in-line with effort of acquiring it.

**Comment by 55Honey** [comment]
> This is an incredible effort
Correct, it's perfectly meant to be like that.

Thanks for your suggestion.
There are no plans to even consider any kind of change to the currently required amount of tokens anytime soon.

**Comment by andovcr** [comment]
@55Honey Any chance this suggestion can be re-opened to provide for further discussion from the community?  Given that another suggestion was closed after 10 minutes and staff have stated they don't wish to do that moving forward?

**Comment by shaszun** [comment]
I will support the staff with this one. I know it's tedious 3/3, but we are talking about a legendary here, which can be transmoged. No other legendary is enabled to do that at the moment (sadly). And with 3/3 you get all other rewards at the same time too. I don't see a problem to get 3 tokens as a hunter or warlock and the other 3 with your caster class. It's hard and brain dead farm, but you get shiny pixels at the end.

I know it's not the main subject for this topic, but it was mentioned. That's why I will say my piece too. Characters trasfer between accounts (same owner) would be a very good opportunity for the server as additional revenue in the shop. I know few people that might use it. Even at the begining of the hardcore challenge after some info got public for the rewards. One person decided to delete a 50+ level 3/3 char, because his account was full...
